app/views/questionnaire.py

```diff
@@ -216,18 +211,20 @@ def get_timeout_continue(eq_id, form_type, collection_id):  # pylint: disable=un
     return 'true'
 
 
-@questionnaire_blueprint.route('session-expired', methods=["GET"])
+@questionnaire_blueprint.route('session-expired', methods=["POST"])
```

Comment:
Tricky, we already have a sign-out page and it carries a different template meaning I can't merge them together